### PR TITLE
Update dependencies and fix well-known format

### PR DESCRIPTION
**Pull request by wistefan**

### CODE REVIEW

**Review by dwendland — APPROVED** [review]

**Review by msn1357 — APPROVED** [review]

**Review by beknazaresenbek — APPROVED** [review]

File: src/main/java/org/fiware/keycloak/model/VCConfig.java [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import lombok.Builder;
 import lombok.Data;
 
+import java.time.Instant;
```
Comment (beknazaresenbek) [inline src/main/java/org/fiware/keycloak/model/VCConfig.java:7]: one unused import was added

File: src/main/java/org/fiware/keycloak/VCIssuerRealmResourceProvider.java [hunk 2]
```diff
@@ -416,7 +463,7 @@ private Response getErrorResponse(ErrorType errorType) {
 
 	/**
 	 * Options endpoint to serve the cors-preflight requests.
-	 *
+	 * <p>
```
Comment (beknazaresenbek) [inline src/main/java/org/fiware/keycloak/VCIssuerRealmResourceProvider.java:466]: probably forgotten part of documentation?

Comment (wistefan) [reply src/main/java/org/fiware/keycloak/VCIssuerRealmResourceProvider.java]: fixed

File: src/main/java/org/fiware/keycloak/model/VCConfig.java (hunk 1 above)
Comment (wistefan) [reply src/main/java/org/fiware/keycloak/model/VCConfig.java]: fixed

**Pull request merged**